internal/kafka/internal/services/data_plane_cluster_test.go

```diff
@@ -676,7 +749,7 @@ func Test_DataPlaneCluster_clusterCanProcessStatusReports(t *testing.T) {
 	}
 }
 
-func TestNewDataPlaneClusterService_GetDataPlaneClusterConfig(t *testing.T) {
+func Test_NewDataPlaneClusterService_GetDataPlaneClusterConfig(t *testing.T) {
```

Comment:
This test name does not seem correct. There is no `NewDataPlaneClusterService` data type. The data type being tested is `dataPlaneClusterService`